boards/BMS/Inc/App/App_Accumulator.h

```diff
@@ -43,6 +65,17 @@ struct Accumulator *App_Accumulator_Create(
  */
 void App_Accumulator_Destroy(struct Accumulator *accumulator);
 
+/**
+ * Get the pack voltage in V
+ * @param accumulator The accumulator to get the pack voltage
+ * @return The accumulator pack voltage in V
+ */
+float App_Accumulator_GetPackVoltage(struct Accumulator *accumulator);
+
+bool App_Accumulator_IsMaxVoltageInRange(struct Accumulator *accumulator);
+
+bool App_Accumulator_IsMinVoltageInRange(struct Accumulator *accumulator);
```

Comment:
can you add documentation for these fxns

Reply:
added